internal/server/events.go

```
@@ -0,0 +1,289 @@
+	w.Header().Set("Content-Type", "text/event-stream")
+	w.Header().Set("Cache-Control", "no-cache")
+	w.Header().Set("Connection", "keep-alive")
+
+	sub := &Subscription{
+		watchPath: watchPath,
+		events:    make(chan eventMsg, 8),
+	}
+	s.events.addClient(sub)
+	defer s.events.removeClient(sub)
+
+	fmt.Fprintf(w, "event: connected\ndata: %s\n\n", toJSON(map[string]string{"type": "connected"}))
```

golangci-lint (2.11.4)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)